### PR TITLE
Convert ImageSpec html file to doxygen

### PR DESCRIPTION
**Pull request by byrnHDF**

### CODE REVIEW

File: doxygen/dox/ImageSpec.dox [hunk 1]
```diff
@@ -0,0 +1,710 @@
...
+or number type that may be used for an Image.
+
+The dataset for an image is distinguished from other datasets by giving
+it an attribute "CLASS=IMAGE". In addition, the Image dataset may
+have an optional attribute "PALETTE" that is an array of object references
+for zero or more palettes. The Image dataset may have additional attributes
+to describe the image data, as defined in \ref subsec_image_spec_spec_attr.
+
+A Palette is an HDF5 dataset which contains color map information.
+A Pallet dataset has an attribute "CLASS=PALETTE" and other attributes
```
Comment (mattjala) [inline doxygen/dox/ImageSpec.dox]: `Pallet dataset` -> `Palette dataset`

File: doxygen/dox/ImageSpec.dox [hunk 2]
```diff
@@ -0,0 +1,710 @@
...
+<td>INTERLACE_MODE</td>
+<td>Optional<sup>3,6</sup></td>
+<td>String</td>
+<td>15</td>
+<td>The layout of components if more than one component per pixel.</td>
+<td>For images with more than one component for each pixel, this optional attribute
+specifies the layout of the data. The values are type #H5T_C_S1 of length
+15. See \ref subsec_image_spec_spec_store for information about the
+storage layout for data.<br />
+<ul><li>INTERLACE_PIXEL (default): the component value for a pixel are contiguous.</li>
```
Comment (mattjala) [inline doxygen/dox/ImageSpec.dox]: `component value` -> `component values`

File: doxygen/dox/ImageSpec.dox [hunk 3]
```diff
@@ -0,0 +1,710 @@
...
+</tr>
+<tr>
+<td>IMAGE_TRANSPARENCY</td>
+<td>Optional<sup>3,5</sup></td>
+<td>Unsigned Integer</td>
+<td></td>
+<td>The index of the transparent color.</td>
+<td>If set, this attribute indicates the index value that should be interpreted
+as the "transparent color". This attribute is HDF5 type #H5T_NATIVE_UINT.
+This attribute may not be used for IMAGE_SUBCLASS="IMAGE_TRUE_COLOR".</td>
```
Comment (mattjala) [inline doxygen/dox/ImageSpec.dox]: `IMAGE_TRUE_COLOR` -> `IMAGE_TRUECOLOR`

File: doxygen/dox/ImageSpec.dox [hunk 4]
```diff
@@ -0,0 +1,710 @@
...
+dimensions: [10][5][3]. For plane interleave, the dataspace would
+be three dimensions: [3][10][5].
+
+In the case of images with only one component, the dataspace may be
+either a two dimensional array, or a three dimensional array with the third
+dimension of size 1. For example, a 5 by 10 image with 8 bit color
+indexes would be an HDF5 dataset with type unsigned 8 bit integer.
+The dataspace could be either a two dimensional array, with dimensions
+[10][5], or three dimensions, with dimensions either [10][5][1] or [1][10][5].
+<p>Image datasets may be stored with any chunking or compression properties
```
Comment (mattjala) [inline doxygen/dox/ImageSpec.dox:405]: Unclosed paragraph tag

File: doxygen/dox/ImageSpec.dox [hunk 5]
```diff
@@ -0,0 +1,710 @@
...
+<tr>
+<td WIDTH="33%">PAL_MINMAXNUMERIC</td>
+<td WIDTH="33%">Optional</td>
+<td WIDTH="34%">Optional</td>
+</tr>
+</table>
+
+\subsection subsec_tab_spec_sect2_23 Storage Layout for Palettes
+The values of the Palette are stored as a dataset. The datatype can
+be any HDF 5 atomic numeric type. The dataset will have dimensions
```
Comment (mattjala) [inline doxygen/dox/ImageSpec.dox]: `HDF 5` -> `HDF5`

**Review by mattjala — DISMISSED** [review]

File: doxygen/dox/ImageSpec.dox (hunk 1 above)
Comment (byrnHDF) [reply doxygen/dox/ImageSpec.dox]: fixed

File: doxygen/dox/ImageSpec.dox (hunk 2 above)
Comment (byrnHDF) [reply doxygen/dox/ImageSpec.dox]: fixed

File: doxygen/dox/ImageSpec.dox (hunk 3 above)
Comment (byrnHDF) [reply doxygen/dox/ImageSpec.dox]: fixed

File: doxygen/dox/ImageSpec.dox (hunk 5 above)
Comment (byrnHDF) [reply doxygen/dox/ImageSpec.dox]: fixed

**Review by lrknox — APPROVED** [review]

**Review by mattjala — APPROVED** [review]

**Pull request merged**